src/annotator/components/test/ClusterToolbar-test.js

```diff
@@ -17,14 +18,36 @@ describe('ClusterToolbar', () => {
       />
     );
 
+  const toggleOpen = wrapper => {
+    act(() => {
+      wrapper
+        .find('button[data-testid="control-toggle-button"]')
+        .simulate('click');
+    });
+    wrapper.update();
+  };
+
   it('renders nothing if the cluster feature is not active', () => {
     const wrapper = createComponent({ active: false });
 
     assert.isEmpty(wrapper.html());
   });
 
+  it('renders collapsed until expanded', () => {
+    const wrapper = createComponent();
+
+    assert.isNotOk(
+      wrapper.find('[data-testid="cluster-style-controls"]').length
+    );
```

Comment:
I'd prefer to use `assert.equal(..., 0)` here and `assert.above(..., 0)` for nicer error messages if they fail.